lsp-protocol.el

```diff
@@ -31,6 +31,17 @@
 (require 'ht)
 (require 's)
 
+(defun lsp-keyword->symbol (keyword)
+  (intern (substring (symbol-name keyword) 1)))
+
+(defun lsp-keyword->string (keyword)
+  (substring (symbol-name keyword) 1))
+
+
+(defun lsp-use-plists ()
+  ;; change to t and recompile lsp-mode to force the usage of plists
+  nil)
```

Comment:
Shouldn't this be `defvar` or `defcustom`?

Reply:
It can be, I changed it for some reason which I cannot remember right now.